packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/ClinicalResultBloodPressure.2.doc.jsx

```diff
@@ -0,0 +1,108 @@
+import TruncatedResult from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressureTruncated';
+import TruncatedResultSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressureTruncated.jsx';
+import InterpretationResults from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressureInterpretation';
+import InterpretationResultsSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressureInterpretation.jsx';
+import SizeChangeResult from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressureChangeSize';
+import SizeChangeResultSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressureChangeSize.jsx';
+import KnownNoData from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressureKnownNoData';
+import KnownNoDataSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressureKnownNoData.jsx';
+import PartialNoData from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressurePartialNoData';
+import PartialNoDataSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressurePartialNoData.jsx';
+import KnownErrorAndPartialError from '../example/clinicalresultbloodpressure/ClinicalResultBloodPressureKnownErrorAndPartialError';
+import KnownErrorAndPartialErrorSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/clinicalresultbloodpressure/ClinicalResultBloodPressureKnownErrorAndPartialError.jsx';
+
+const DocPage = () => (
+  <DocTemplate
+    packageName={name}
+    readme={ReadMe}
+    srcPath={`https://github.com/cerner/terra-clinical/tree/master/packages/${name}`}
+    examples={[
+      {
+        title: 'Default Blood Pressure Clinical Result',
+        example: <DefaultResult />,
+        source: DefaultResultSrc,
+      },
+      {
+        title: 'Modified Blood Pressure Clinical Result with Comments',
+        description: '',
+        example: <DecoratedResult />,
+        source: DecoratedResultSrc,
+      },
+      {
+        title: 'Unverified Blood Pressure Result with Hide Unit of Measure',
+        description: '',
+        example: <UnverifiedResult />,
+        source: UnverifiedResultSrc,
+      },
+      {
+        title: 'Clinical Blood Pressure Result with additional displays',
+        description: '',
+        example: <ExtraDisplaysResult />,
+        source: ExtraDisplaysResultSrc,
+      },
+      {
+        title: 'Blood Pressure Clinical Results with Intrepretation',
+        description: '',
+        example: <InterpretationResults />,
+        source: InterpretationResultsSrc,
+      },
+      {
+        title: 'Changing the Blood Pressure Clinical Result Size',
+        description: '',
```

Comment:
These empty description keys can be removed

Reply:
Removed https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-e9fba86f509874f216a233609076ac1fL44